src/o365/spo/commands/file/file-add.spec.ts

```diff
@@ -31,56 +31,56 @@ describe(commands.FILE_ADD, () => {
 
```

Comment:
In the future, let's try to avoid reformatting existing files, as it makes it hard to find out what's changed exactly.

Reply:
Agreed, sorry about that, I didn't realise until I pushed it 😬